package.json

```diff
@@ -16,6 +16,10 @@
   "author": "Todd Bluhm",
   "license": "MIT",
   "homepage": "https://github.com/standard/ts-standard",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Maytha8/ts-standard.git"
```

Comment:
Is that a mistake? 😄

Reply:
Yep, I had it set to my fork so I could test commitlint, but it looks like I forgot to change it. This package.json property is needed as part of commitlint's configuration, so it can create releases automatically on the correct GitHub repo.